styles/orgsync-widgets.scss

```diff
@@ -187,4 +188,11 @@
       }
     }
   }
+
+  .osw-blank-slate-message{
```

Comment:
You did so well with all of these files, and then you missed a space. (feelsbadman)